common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1,4 +1,32 @@
 {
+  "AGGREGATE_FILTER_EXPRESSION_ERROR" : {
```

Comment:
The `_ERROR` word is useless because all error classes in the files are related to errors. Let's remove the word, and assign shorted and "conditional" name like `INVALID_AGGREGATE_FILTER`

Reply:
Done as suggested.